locales/sk/amo_description.html

```diff
@@ -0,0 +1,28 @@
+Cast&nbsp;Kodi, pridá možnosť posielania videí a hudby do <strong>Kodi</strong>:
+<ul>
+  <li>
+    priame odkazy: <em>avi</em>, <em>mkv</em>, <em>mp3</em>, <em>flac</em>, ... a torrent / magnet;
+  </li>
+  <li>
+    YouTube, Twitch, Vimeo, SoundCloud a tiež Ace&nbsp;Stream, Apple&nbsp;Podcasty, Bigo&nbsp;Live, BitChute, Blog&nbsp;Talk&nbsp;Radio, Castbox, Dailymotion, DevTube, Facebook, Flickr, Full30, Instagram, Jamendo, LBRY, LiveLeak, Metacafe, Mixcloud, My&nbsp;Cloud&nbsp;Player, Overcast, PeerTube, Pippa, podCloud, PodMust, Radio, Radioline, Steam, Streamable, TikTok, Ultimedia, Veoh, VideoPress, Viously, YT&nbsp;Home;
+    <ul>
+      <li>Nemecko: Arte;</li>
+      <li>Belgicko: VRT&nbsp;NU;</li>
+      <li>USA: KCAA&nbsp;Radio;</li>
+      <li>Francúzsko: 20&nbsp;Minutes, AlloCiné, Arte, Arte&nbsp;Radio, France&nbsp;Inter, Futura Sciences, Gamekult, JeuxVideoCom, Konbini, Le&nbsp;Point, L'Internaute, Melty, Ouest-France;</li>
+      <li>Irán: آپارات;</li>
+      <li>Island: Útvarp&nbsp;Saga;</li>
+      <li>Holandsko: Dumpert;</li>
+      <li>Rusko: Первый&nbsp;канал;</li>
+      <li>Spojené kráľovstvo: Daily&nbsp;Mail, The&nbsp;Guardian.</li>
+    </ul>
+  </li>
+</ul>
+Cast&nbsp;Kodi tiež analyzuje stránky a hľadá v nich odkazy na videá a hudby z externých platforiem. Ak napríklad stránka obsahuje YouTube video, toto video bude poslané do Kodi.
+
+Pre poslanie videa/hudby môžete do kontextovej ponuky odkazu / stránky / výberu pridať nasledovné 3 položky: <em>Prehrať teraz v Kodi</em>, <em>Prehrať neskôr v Kodi</em> a <em>Pridať do poradia v Kodi</em>. Prítomné je aj diaľkové ovládanie (dostupné na lište s nástrojmi), ktoré vám umožní odoslať aktuálnu kartu do Kodi a pozastaviť video, prejsť na ďalšiu položku, upraviť hlasitosť, ...
+
+
+V Kodi musíte <em>Povoliť diaľkové ovládanie aplikáciam na iných systémoch</em>. Túto možnosť nájdete v <em>Nastavenia</em> / <em>Servery</em> / <em>Ovládanie</em>.
```

Comment:
Kodi uses [_Povoliť diaľkové ovládanie pomocou programov v iných systémoch_](https://github.com/xbmc/translations/blob/master/kodi-translations/kodi-main/merged-langfiles/kodi.core/resource.language.sk_sk/resources/strings.po#L2802) for _Allow remote control by programs on other systems_.

```suggestion
V Kodi musíte <em>Povoliť diaľkové ovládanie pomocou programov v iných systémoch</em>. Túto možnosť nájdete v <em>Nastavenia</em> / <em>Servery</em> / <em>Ovládanie</em>.
```

Reply:
It's pretty much the same. Kodi's translation sounds more like "allow remote control by using (with help of) programs on other systems" which I found not too accurate though.